### ISSUE TITLE
Modify EEG Conformer for regression

### ISSUE THREAD

**Issue by ZaraCook** [ZaraCook/eegLLM#5]
- Change the output layer to produce a single continuous value (age prediction) instead of class probabilities
- Adjust the loss function from cross-entropy to mean squared error (MSELoss)
- Modified model hyperparameters, including adjusting the number of filters, attention mechanisms, and dropout rates
